Automattic/jetpack
# Jetpack Editor Sidebar: Add a way to open the sidebar based on query param

Fixes #21155

#### Changes proposed in this Pull Request:

* Adds a function to /extensions/shared/jetpack-plugin-sidebar.js to open the Jetpack plugin sidebar if the URL includes a query param `jetpackSidebarIsOpen` set to `true`

#### Jetpack product discussion



This is an initial step to bring a Publicize link to wp-admin via the Jetpack plugin. With this PR we can link to the editor with the Publicize panel open. See pd2C0z-ao-p2

#### Does this pull request change what data or activity we track or use?



Nope.

#### Testing instructions:






* Open a post in the editor and make sure the Jetpack sidebar is closed.
* Add `?jetpackSidebarIsOpen=true` to the URL and hit enter
* The sidebar should open when the editor loads:
<img width="294" alt="Screen Shot 2021-09-22 at 1 57 08 PM" src="https://user-images.githubusercontent.com/1689238/134396588-d25e7c0c-caa0-4e42-8ab1-4a1ced5d5fcb.png">

 * Close the sidebar and remove the query param from the URL.
 * The sidebar should stay closed when the editor loads.
